tasks/configure.yml

```diff
@@ -54,6 +54,11 @@
   when: (item.path | basename) not in haproxy_frontends_file_list
   with_items: "{{ directory_contents.files }}"
 
+- name: 'Validate frontends'
+  ansible.builtin.fail:
+    msg: "Frontends description use undefined variable : {{ haproxy_frontends }}"
```

Comment:
I would suggest removing the pair of mustache otherwise you will receive a run-time error about this undefined variable _haproxy_frontends_ 😉

Reply:
There is a misunderstanding. It's not haproxy_frontends that might be undefined. If it's undefined the when clause should protect from that case.
We want to inform the user that in the content built into haproxy_frontends there might be a variable used that is not defined.